bool EMERGENCY_EXIT_HAPPENED
bool READY_TO_START
bool LAND_DRONE